spec/controllers/user_controller_spec.rb

```diff
@@ -0,0 +1,54 @@
+# spec/controllers/users_controller_spec.rb
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
```

Comment:
- [ ] Just as I suggested above, these tests are of `type` request and not controller, I'll suggest you update the types to match the correct type of test cases here. eg: `RSpec.describe 'Users', type: :request do` :smile: :small_airplane: